pages/portfolio.html

```diff
@@ -4,9 +4,55 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet"/>
+  <link href="../styles/portfolio.css" rel="stylesheet"/>
+  <title>Portfolio</title>
 </head>
 <body>
-  
+  <header>
+    <img src="/assets/in-the-snow.jpg" alt="A woman in the snow" />
```

Comment:
Did this path load the image for you? On my end, with your project as is, I had to add `..` in front of the relative path to see your pics (like you did on lines 7-8 for the href attributes for the css files, which were correctly linked)